tests/test_converter.py

```diff
@@ -1775,10 +1816,137 @@ def test_mp_to_pauli(self, measurement_type, observable, obs_string, wire, regis
         # remaining wires are all Identity
         assert np.all([op == "I" for op in pauli_op_list])
 
+    @pytest.mark.parametrize("measurement_type", [qml.expval, qml.var])
+    @pytest.mark.parametrize("register_size", [3, 5, 7])
+    @pytest.mark.parametrize(
+        "observables, coeffs",
```

Comment:
We should test it for multi-qubit observables as well.

Reply:
Actually that was the original test back when the test only allowed for single qubit observables. This type of test doesn't work on multi-qubit observables. Probably better to just compare to manual test cases every time.